### DISCUSSION TITLE
Mapbox v11 - NEW 3D ENVRONMENTS - react-native

### DISCUSSION THREAD

**Discussion by fabioabreuit** [rnmapbox/maps#3220] [Ideas]
## New Feature
It's possible update and use a mapbox v11 in react-native using a rnmapbox/maps?

I saw that there are some old posts about it, but no conclusion, any progress?

## Why
NEW 3D ENVIRONMENTS ENHANCE WAYFINDING AND LOCATION AWARENESS

![mapbox v11](https://github.com/rnmapbox/maps/assets/52165013/424932bd-148f-4d2c-b6f1-8ff0ea629776)

**Comment by mfazekas** [comment]
You should be able to us using 10.1.* betas.

See V11 sections on our setup guides:
https://github.com/rnmapbox/maps/blob/main/ios/install.md#v11-support

**Comment by fabioabreuit** [comment]
Nice, thanks
